test/test_modelFiles.m

```diff
@@ -0,0 +1,37 @@
+function test_modelFiles()
```

Comment:
This is a very ugly test but it might let us know when we get some weird failure when the json model files are not properly formated